src/torchcodec/_core/SingleStreamDecoder.h

```diff
@@ -493,6 +492,7 @@ class SingleStreamDecoder {
   SeekMode seekMode_;
   ContainerMetadata containerMetadata_;
   UniqueDecodingAVFormatContext formatContext_;
+  std::unique_ptr<DeviceInterface> deviceInterface;
```

Comment:
Class variable members end with an underscore (`foo_`).

Reply:
ah, I overlooked moving it from the struct. Will fix in a follow up PR.